src/api/todos.ts

```diff
@@ -0,0 +1,22 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 1826;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
```

Comment:
`/todos` repeats in all these functions, let's move `/todos` into const

Reply:
I have prop isLoading in TodoItem,so that i give name for hooks IsDeleting